### PR TITLE
Suse Theme updated for new brand

### LINKED ISSUE #385: Update theme to reflect new SUSE branding

### PR DESCRIPTION
**Pull request by nwmac**
Fixes #385

### CODE REVIEW

File: src/frontend/packages/suse-theme/package.json [hunk 1]
```diff
@@ -1,17 +1,19 @@
 {
-  "name": "@suse/theme",
-  "version": "0.0.1",
+  "name": "@susestratos/theme",
```
Comment (richard-cox) [inline src/frontend/packages/suse-theme/package.json:2]: The org name of suse-extensions should also be updated

File: src/frontend/packages/suse-theme/loader/loading.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .stratos-initial-load-spinner-container {
   align-items: center;
-  background-color: #0D2C40;
+  background-color: #f2f2f2;
```
Comment (richard-cox) [inline src/frontend/packages/suse-theme/loader/loading.css:3]: The background for the loading page I think should be the same colour as the background of the log in/loading page
- transition wise it goes pre-loading page light grey --> log in/loading page dark green
- if user is logged in it then does a third colour hope light grey --> dark green --> light grey (most logged in pages)
- the pre-loading page is only around for a short time so can flicker, particularly jarring in the pre-login --> login --> logged in scenario

File: src/frontend/packages/suse-theme/package.json [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
+  "version": "1.0.0",
```
Comment (richard-cox) [inline src/frontend/packages/suse-theme/package.json]: Should suse-extensions also be 1.0.0 for the initial release?

**Review by richard-cox — CHANGES_REQUESTED** [review]
Two unrelated comments, one suggestion

File: src/frontend/packages/suse-theme/package.json (hunk 3 above)
Comment (nwmac) [reply src/frontend/packages/suse-theme/package.json]: 0.0.0 as a placeholder

**Review by richard-cox — APPROVED** [review]

**Pull request merged**